src/cache/cache_las.c

```diff
@@ -665,20 +572,28 @@ int
 __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MULTI *multi)
 {
     WT_CACHE *cache;
+    WT_DECL_ITEM(full_value);
     WT_DECL_ITEM(key);
     WT_DECL_ITEM(mementos);
+    WT_DECL_ITEM(modify_value);
+    WT_DECL_ITEM(prev_full_value);
+    WT_DECL_ITEM(tmp);
     WT_DECL_RET;
     WT_KEY_MEMENTO *mementop;
+#define MAX_REVERSE_MODIFY_NUM 16
```

Comment:
Here I follow mongodb's usage, which limits the maximum number of modifies in a single MODIFY update to be 16.

Reply:
I'd toss in a comment that notes this isn't a fatal problem if it's exceeded, just means we won't use modify records.